nautobot/extras/urls.py

```diff
@@ -166,6 +169,10 @@
         name="configcontextschema_notes",
         kwargs={"model": ConfigContextSchema},
     ),
+    # contacts
+    path("contact-associations/add-new-contact/", views.ObjectNewContactView.as_view(), name="object_contact_add"),
+    path("contact-associations/add-new-team/", views.ObjectNewTeamView.as_view(), name="object_team_add"),
+    path("contact-associations/assign-contact-team/", views.ObjectAssignView.as_view(), name="object_assign"),
```

Comment:
Probably would want to rename this to `object_contact_team_assign` or similar as `object_assign` is a bit vague?